databook/glossary.md

```diff
@@ -18,6 +18,18 @@ CSV
 Ephys
   Shorthand for electrophysiology.
 
+Fast spiking neuron
```

Comment:
unit?

Reply:
In the context of intracellular recordings these are neurons, however I said in the context of extracellular recordings "units are" (etc...).